src/assetbundles/websitetoolboxforum/dist/js/Websitetoolboxforum.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Website Toolbox Forum plugin for Craft CMS
+ * Website Toolbox Forum JS
+ * @author    Website Toolbox 
+ * @copyright Copyright (c) 2020 Website Toolbox
+ * @link      https://websitetoolbox.com/
+ * @package   Website Toolbox Forum
+ * @since     3.0.0
+ */
+ function getCookie(){
+ 	var i,x,y,ARRcookies  =	document.cookie.split(";");
+    for (i=0;i<ARRcookies.length;i++) {
+        x = ARRcookies[i].substr(0,ARRcookies[i].indexOf("="));
+        y = ARRcookies[i].substr(ARRcookies[i].indexOf("=")+1);
+        x = x.replace(/^\s+|\s+$/g,"");
+        if (x=='forumLogoutToken') {
+            var forumToken = y;          
+        }
+    }
+    return forumToken;
```

Comment:
Please correct the indentation.

Reply:
It was showing correct when I tried to edit here in GIT. But still saved one more time. Its showing correct now.